superset/assets/src/dashboard/util/charts/getFormDataWithExtraFilters.js

```diff
@@ -74,7 +110,9 @@ export default function getFormDataWithExtraFilters({
     extra_filters: getEffectiveExtraFilters({
       dashboardMetadata,
       filters,
-      sliceId
+      sliceId,
+      filterKeys,
+      linkedSlicesExistInFilters,
```

Comment:
if filterKeys is  const list of values then we can skip it as args and refer as const vis import
rename it as TIME_FILTER_KEYS

Reply:
done